crates/cast/bin/main.rs

```diff
@@ -216,6 +217,28 @@ async fn main_args(args: CastArgs) -> Result<()> {
             let decoded_event = event.decode_log_parts(None, &hex::decode(data)?, false)?;
             print_tokens(&decoded_event.body);
         }
+        CastSubcommand::DecodeError { sig, data } => {
+            let error = if let Some(err_sig) = sig {
+                get_error(err_sig.as_str())?
+            } else {
+                let data = data.strip_prefix("0x").unwrap_or(data.as_str());
+                let selector = &data[..8];
+                let err = SignaturesIdentifier::new(Config::foundry_cache_dir(), false)?
+                    .write()
+                    .await
+                    .identify_error(&hex::decode(selector)?)
+                    .await;
+                if err.is_none() {
```

Comment:
let Some ... else

Reply:
changed in https://github.com/foundry-rs/foundry/pull/9428/commits/1834215d256f0afb1d574ae538c03b67d6ed5eb1